docs/semgrep-app/notifications.md

```diff
@@ -92,7 +92,7 @@ semgrep:
   script:
     # SEMGREP_APP_TOKEN can be obtained from semgrep.dev/manage/settings.
     # The SEMGREP_APP_TOKEN should be treated like a secret and not hard-coded into your code.
-    - semgrep-agent --publish-token $SEMGREP_APP_TOKEN
+    - semgrep-agent --publish-token $SEMGREP_APP_TOKEN TODO
```

Comment:
How will the token look like now? - `semgrep ci --publish-token $SEMGREP_APP_TOKEN`?

@brendongo @underyx

Reply:
We removed this flag! Now it's enough to have the `SEMGREP_APP_TOKEN` environment variable set and it will automatically be picked up. In fact, that was the case even with `semgrep-agent`.

Reply:
Updated. TODO note for myself - test the procedure given in this section.

Reply:
Procedure tested. Seems to work. Additional questions will be left for another comment for the sake of clarity after updates. Resolving comment.